src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -160,6 +182,88 @@ private void SetPanelSizes(float panel1Fraction)
         _panel2Size = string.Create(CultureInfo.InvariantCulture, $"{(1 - panel1Fraction):F3}fr");
     }
 
+    public async Task OnPageKeyDownAsync(KeyboardEventArgs args)
+    {
+        if (_splitterRef is null || !args.ShiftKey)
+        {
+            return;
+        }
+
+        if (args.Key.Equals("t", StringComparison.InvariantCultureIgnoreCase))
+        {
+            await HandleToggleOrientation();
+            return;
+        }
+
+        if (args.Key.Equals("o", StringComparison.InvariantCultureIgnoreCase) && SelectedValue is not null)
+        {
+            _internalShowDetails = !_internalShowDetails;
+            await InvokeAsync(StateHasChanged);
+            return;
+        }
+
+        var hasChanged = false;
+
+        if (args.Key.ToLower() == "r")
+        {
+            ResetPanelSizes();
+            hasChanged = true;
+        }
+
+        GetPanelSizes(_splitterRef.Panel1Size, _splitterRef.Panel2Size, out var panel1Size, out var panel2Size, out var panel1Fraction);
+
+        if (panel1Size is null || panel2Size is null || panel1Fraction is null)
+        {
+            return;
+        }
+
+        if (args.Key is "+")
+        {
+            SetPanelSizes(panel1Fraction.Value - 0.05f);
+            hasChanged = true;
+        }
+        else if (args.Key is "-" or "_")
```

Comment:
I'm a little thrown off by `Shift` + `-` given that that makes it a `_`. But I think its probably ok since its consistent with browser +/- for zoom (which uses ctrl the same way) and you made it work with the numpad +/- too.